### ISSUE TITLE
Thanks for your submission!

### ISSUE THREAD

**Issue by chug2k** [stevenkeezer/product-pricing-page#1]
Thank you for your submission. Bootstrap is such a wonderful tool that helps you get things done quickly and easily, and you are one step closer to mastering Bootstrap with this assignment.

Building up a nice Pricing Page shows you how to build up Form quickly (which you will have to do many times as a developer (think of login page)). You also learn about the Bootstrap Grid System, which makes laying out elements such a breeze. 

## High Level Comments

You got the general page layout correct, and the responsive navbar! Great work. 

Your code indentation is good. Many newcomers overlook this, but spending more time making sure you `divs` align nicely today could save you hours of debugging time in the future. You have a few weird ones in your CSS file, and watch out for missing semicolons in CSS:

```
.icons {
  display: flex;
  justify-content: center;
  padding-top: 10%
}
```

Nice job with the media query.

You could make use of [Prettier Plugin](https://prettier.io/) in VS Code to help you too. Pressing `Ctrl + Shift + P`, and choosing `Format Document` help aligining your code on the fly. If you see `Prettier` Plugin fails to align your code, there is a  likely chance you forgot to close your `div` somewhere.  

## Things to Improve

We should mention this in class, but unfortunately you can't just drop in 'Avernir Next' into a webpage - the main reason being that it doesn't display correctly if the end user doesn't have Avernir Next installed on her system. I see a weird default font for that reason: 

![Screenshot 2019-09-26 at 9 52 40 PM](https://user-images.githubusercontent.com/44512/65699255-02f13300-e0a8-11e9-9a54-7bad3631c040.png)

The correctly use a font, try something like Google Fonts at https://fonts.google.com/. 

Please have your `Readme.md` file to describe your Homework Project, and laying out nicely all the User Stories. This helps mark you as a professional who cares about every aspect of his work, which lays a great impression on potential employers who may take a look on your Github repository.

Also, try deploying your page on Netlify, as it helps people to visualize your code better too :)

**Comment by stevenkeezer** [comment]
Thank you for taking the time to review my code,  Charles. This feedback is
very valuable to me. I am a big fan of Avenue Next for its readability. I
didn't realize this was contingent upon my local machine.

I admit,  I have overlooked Readme files as an afterthought. I will spend
more time describing my projects for the public and future employers to
understand my repository after each class.

I actually have deployed my project on netlify, I didn't know if that was
the link I should have shared, you can view it here:
https://trusting-newton-8dbe50.netlify.com/

This is the first time I have deployed anything and I was very proud of
that. I  was never able to deploy my flask app.

On some rather unfortunate news, I lost my wallet today. I am sending a new
debit card to the address of coderschool. It should be arriving by next
Friday. I hope this is okay with you.

Thank you again for the feedback Charles,  I am a big fan out what you are
doing here, and I am learning a lot in this program.

**Comment by chug2k** [comment]
Yeah, just link the netlify repository. Sorry for not giving a README template, we've done that for the the group project. 

BUMMER about the wallet. That's totally inconvenient. In the meantime, if you need to borrow a little bit of cash, just let me know. I have a lot of sympathy for wallet losers because I've done that multiple times. 

Thanks for the kind words, glad to see you rocking it.

**Comment by stevenkeezer** [comment]
Thank you Charles. I have some cash stowed away. See you tomorrow.
